### PR TITLE
VPC DNS and Elasticache support

### PR DESCRIPTION
**Pull request by kewubenduben**
I hope this helps cleaning up a bit.

### CODE REVIEW

**Comment by e0d** [comment]
@kewubenduben this is great.  Have you signed a contributor  agreement?

**Comment by kewubenduben** [comment]
@e0d, where do I sign that?

**Comment by e0d** [comment]
http://code.edx.org/individual-contributor-agreement.pdf

**Comment by kewubenduben** [comment]
https://s3.amazonaws.com/uploads.hipchat.com/67425/471578/e0gqyvbldu5f7w9/individual-contributor-agreement.pdf

File: cloudformation_templates/edx-reference-architecture.json [hunk 1]
```diff
@@ -320,6 +320,8 @@
   "Resources":{
     "EdxVPC":{
       "Type":"AWS::EC2::VPC",
+      "EnableDnsSupport" : "true",
```
Comment (feanil) [inline cloudformation_templates/edx-reference-architecture.json:323]: These two properties actually belong in the properties hash below.  Can you please move them there?

**Comment by jarv** [comment]
closing this since we haven't gotten a response

**Pull request closed without merging**